### DISCUSSION TITLE
升级到最新版本之后，打包markdown 报错了

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [umijs/dumi#2106] [Q&A]
info  - dumi v2.3.4
info  - Umi v4.2.0

jia检查了下相应的文件，没什么问题，或者哪里可以关掉ts 的报错
![image](https://github.com/umijs/dumi/assets/15354455/ead6f029-d500-4c93-83ad-2b4258efe579)

**Comment by Chasen-Zhang** [comment]
![image](https://github.com/umijs/dumi/assets/15354455/c11891f9-ddcf-4d17-b02e-d2d0e935649c)
这个版本没问题，是做了什么严格的限制吗

**Reply by jeffwcx** [reply]
能给个可复现的demo吗？

**Comment by PeachScript** [comment]
`.jsx` 文件里写了 TypeScript 导致的报错，要写 TS 需要用 `.tsx`

**Reply by Chasen-Zhang** [reply]
我这个是markdown 文件啊，是.md后缀的文件

**Comment by GuoDapeng** [comment]
咱俩好想遇到了类似的问题，https://github.com/umijs/dumi/issues/2112 我之前混淆了 `jsx` 和 `tsx` ，更新之后，这个解析变严格了。
